dash/backend/src/modules/falco/dao/falco.dao.ts

```diff
@@ -139,41 +239,52 @@ export class FalcoDao {
 
+        // for some reason, the returned query never makes it back. it hits the return statement then gets lost somewhere
+        // let query = await this.buildFalcoLogQuery(knex, clusterId, priorities, orderBy, startDate, endDate, namespace, pod, image);
+
         let query = knex.select()
-            .from('project_falco_logs')
-            .where('cluster_id', clusterId);
+          .from('project_falco_logs')
+          .where('cluster_id', clusterId);
 
         if (priorities) {
             query = query.whereIn('level', priorities);
         }
-        if (orderBy == 'Priority Desc' || orderBy =='Priority Asc' ||  orderBy =='Date Desc'||  orderBy =='Date Asc' ||  orderBy == null ||  orderBy == undefined) {
+        const relevantOrderBys = [
+            'Priority Desc',
+            'Priority Asc',
+            'Date Desc',
+            'Date Asc',
+            null,
+            undefined,
+        ];
+        if (orderBy in relevantOrderBys) {
             switch (orderBy) {
                 case 'Priority Desc':
                     query = query.orderByRaw(
-                        'CASE ' +
-                        ' WHEN level = \'Emergency\' then 1' +
-                        ' WHEN level = \'Alert\' then 2' +
-                        ' WHEN level = \'Critical\' then 3' +
-                        ' WHEN level = \'Error\' then 4' +
-                        ' WHEN level = \'Warning\' then 5' +
-                        ' WHEN level = \'Notice\' then 6' +
-                        ' WHEN level = \'Informational\' then 7' +
-                        ' WHEN level = \'Debug\' then 8' +
-                        'END'
+                      'CASE ' +
+                      ' WHEN level = \'Emergency\' then 1' +
+                      ' WHEN level = \'Alert\' then 2' +
+                      ' WHEN level = \'Critical\' then 3' +
+                      ' WHEN level = \'Error\' then 4' +
+                      ' WHEN level = \'Warning\' then 5' +
+                      ' WHEN level = \'Notice\' then 6' +
+                      ' WHEN level = \'Informational\' then 7' +
+                      ' WHEN level = \'Debug\' then 8' +
+                      'END'
                     );
                     break;
                 case 'Priority Asc':
                     query = query.orderByRaw(
-                        'CASE ' +
-                        ' WHEN level = \'Debug\' then 1' +
-                        ' WHEN level = \'Informational\' then 2' +
-                        ' WHEN level = \'Notice\' then 3' +
-                        ' WHEN level = \'Warning\' then 4' +
-                        ' WHEN level = \'Error\' then 5' +
-                        ' WHEN level = \'Critical\' then 6' +
-                        ' WHEN level = \'Alert\' then 7' +
-                        ' WHEN level = \'Emergency\' then 8' +
-                        'END'
+                      'CASE ' +
```

Comment:
ugh auto-formatting the file? :D

Reply:
I had tried to rework it and "undo" didn't undo the prettifying ;)

Reply:
its nbd.